benchmarking.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>This line gave me an errer because I do not have this directoy</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>this should be now fixed</p>